### ISSUE TITLE
Sign in button on ahoy.hack.club won't let you authorize

### ISSUE THREAD

**Issue by uniquepersun** [hackclub/high-seas#959]
the uri doesn't matches 
"redirect_uri did not match any configured URIs. Passed URI: https://ahoy.hack.club/api/slack_redirect"

**Comment by Gitstar-OC** [comment]
Hey, could you provide more info I will look into it then. While it looks like ahoy.hack.club is not a part of this codebase to be solved.

**Comment by uniquepersun** [comment]
before couple days, highseas used to give referral link like this(highseas.hackclub.com/?ref=slackid)
But now it gives this type: https://ahoy.hack.club/1773

Which upon visiting do not redirects to highseas.hackclub.com instead become like this(https://ahoy.hack.club/?ref=U078GJ63AQ0)

and at that point if click already in slack button, it won't let you sign in..


Ig it is also cuz the user is not supposed to click on that button since that guy have to put in their email and start sign up, but still it's a issue

**Comment by Gitstar-OC** [comment]
Wait

**Comment by taciturnaxolotl** [comment]
Hmm this is a good issue; I added that as a valid redirect url but now you are just stuck on the home page. Ill bring it up in the high seas engineering channel!

**Comment by uniquepersun** [comment]
Hii krn, since this page is for new comers and they being stuck on that page isn't good.. I think redirecting it back to the high seas link will be better. (May be record the refferal upon clicking/entering the email then just redirect and handle logging in on the main highseas page)
is the engineering channel private?

**Comment by taciturnaxolotl** [comment]
I agree; im just not sure the hosting situation

**Comment by uniquepersun** [comment]
Ohh okay 👍🏻

**Comment by maxwofford** [comment]
heads up, ahoy.hack.club is only used when referring, and we don't give a referral bonus for people in the slack– just new users who type in an email. I'm up to fix this, but it sounds like maybe there's a higher level issue than just redirects.

**Comment by taciturnaxolotl** [comment]
Agreed; i think it might be helpful to have a banner telling you who referred you and removing the slack button completely

**Comment by uniquepersun** [comment]
**"and we don't give a referral bonus for people in the slack"** that ik and yeaa removing that or making it redirect to the main page would be awesome
